Documentation/MigrateToComposer/Requirements.rst

```diff
@@ -64,8 +88,50 @@ a later step. We will use :file:`public/` in our example.
 ..  todo: What does refactor concrete mean?
 
 If you do not have such a web root directory, you will have to refactor your
-project before proceeding. Please be aware that you may need to tell your web
-server about the changed web root folder if necessary.
+project before proceeding. First, you create the new directory :file:`public` and
+basically move everything you have inside that subdirectory. Then check all
+of your custom code for path references that need to be adjusted to contain
+the extra :file:`public/` part inside of it. Usually, HTTP(S) links are relative
+to your root, so only absolute path references may need to be changed (e.g. cronjobs,
+CLI references, configuration files, :file:`.gitignore`, ...).
+
+Please be aware that you very likely need to tell your web
+server about the changed web root folder if necessary. You do that by changing a
+`DocumentRoot` (Apache) or `root` (Nginx) configuration option. Most hosting
+providers offer a user interface to change the base directory of your project.
+
+Also for local development with DDEV or Docker you will need to adjust the
+corresponding configuration files.
+
+Git version control, local development and deployment
+=====================================================
+
+This migration guide expects that you are working locally with your project and use
+Git version control for it.
+
+If you previously used the TYPO3 Legacy installation (from a release ZIP) and did
+not yet use Git versioning, this is a good time to learn about version control first.
+
+All operations should ideally take place in a separate branch of your Git repository.
```

Comment:
Too big step: separate branch -> deploy to staging/production. There is missing something in between.